### PR TITLE
Genomes and Super Studies

### PR DESCRIPTION
**Pull request by mberacochea**

### CODE REVIEW

File: emgapi/models.py [hunk 1]
```diff
@@ -640,12 +640,110 @@ class Meta:
...
+                Prefetch('biome', queryset=Biome.objects.all()),
+            )
+        return queryset
+
+
+class SuperStudy(models.Model):
+    """
+    Aggregation of studies.
+    Each Super Study will have multiples Studies under 2 categories:
+    - Flaship Projects, those that are directly related to the Super Study
```
Comment (mscheremetjew) [inline emgapi/models.py]:
```suggestion
    - Flagship Projects, those that are directly related to the Super Study
```

File: emgapi/filters.py [hunk 2]
```diff
@@ -233,6 +233,27 @@ def value_from_datadict(self, data, files, name):
...
 
 
+class SuperStudyFilter(django_filters.FilterSet):
+
+    biome_name = django_filters.CharFilter(
+        method='filter_biome_name', distinct=True,
+        label='Biome name',
+        help_text='Biome name')
+
+    def filter_biome_name(self, qs, name, value):
```
Comment (mscheremetjew) [inline emgapi/filters.py]:
```suggestion
    def filter_biome_lineage(self, qs, name, value):
```

File: emgapi/filters.py [hunk 3]
```diff
@@ -233,6 +233,27 @@ def value_from_datadict(self, data, files, name):
         return list(set(ret))
 
 
+class SuperStudyFilter(django_filters.FilterSet):
+
+    biome_name = django_filters.CharFilter(
+        method='filter_biome_name', distinct=True,
```
Comment (mscheremetjew) [inline emgapi/filters.py]:
```suggestion
        method='filter_biome_lineage', distinct=True,
```

File: setup.py [hunk 4]
```diff
@@ -7,7 +7,7 @@
 _requirements = os.path.join(_base, 'requirements.txt')
 _requirements_test = os.path.join(_base, 'requirements-test.txt')
 
-version = "1.4.4"
+version = "1.4.5"
```
Comment (mscheremetjew) [inline setup.py]: v1.4.5 and v1.4.6 do already exist! Also I was wondering with all those changes if we should make it a minor version upgrade to 1.5.0. What do you think?

Comment (mscheremetjew) [inline setup.py]: v1.4.5 and v1.4.6 do already exist! Also I was wondering with all those changes if we should make it a minor version upgrade to 1.5.0. What do you think?

Comment (mberacochea) [reply setup.py]: Yes I agree, 1.5.0 is the way to go.

Comment (mberacochea) [reply setup.py]: I change the version on a specific commit and add a tag to make it clear.

**Pull request merged**